src/jit/morph.cpp

```diff
@@ -7435,7 +7489,17 @@ bool Compiler::fgCanFastTailCall(GenTreeCall* callee)
         }
         else
         {
+#ifdef _TARGET_ARM_
+            if (varTypeIsFloating(argx))
+            {
+                return false;
```

Comment:
Please add logging to this return using reportFastTailCall decision.

Reply:
Also adding some sort of NYI or comment here explaining why this returns false would be nice.